Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj

```diff
@@ -12,6 +12,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
   <PropertyGroup>
+    <!--PackageId>Philips.CodeAnalysis.MaintainabilityAnalyzers.Dogfood</PackageId-->
```

Comment:
Debug leftover ?

Reply:
Yes. I intentionally left it there (commented out) to ease local development/testing.